tests/integrations/tests/integrations/test_bedrock.py

```
@@ -0,0 +1,300 @@
+                }
+            }
+        })
+        
+    return {"tools": bedrock_tools}
+
+
+class TestBedrockIntegration:
+    """Test suite for Bedrock integration covering core scenarios"""
+
+    @skip_if_no_api_key("bedrock")
+    def test_01_text_completion_invoke(self, bedrock_client, test_config):
```

Ruff (0.14.5)

Unused method argument: `test_config`

(ARG002)